### ISSUE TITLE
What loss function do you use?

### ISSUE THREAD

**Issue by BigDataHa** [javier2491/FaceQnet#17]

**Comment by javier-hernandezo** [comment]
I use the MSE loss integrated in Keras.
